entity-framework/efcore-and-ef6/porting/index.md

```diff
@@ -1,61 +1,71 @@
 ---
-title: Porting from EF6 to EF Core - EF
-description: General information on porting an application from Entity Framework 6 to Entity Framework Core
-author: ajcvickers
-ms.date: 10/27/2016
+title: Port from EF6 to EF Core - EF
+description: A detailed guide to port your 
+author: jeremylikness
+ms.alias: jeliknes
+ms.date: 10/25/2021
 uid: efcore-and-ef6/porting/index
 ---
-# Porting from EF6 to EF Core
+# Port from EF6 to EF Core
 
-Because of the fundamental changes in EF Core we do not recommend attempting to move an EF6 application to EF Core unless you have a compelling reason to make the change.
-You should view the move from EF6 to EF Core as a port rather than an upgrade.
+Entity Framework Core, or EF Core for short, is a total rewrite of Entity Framework for modern application architectures. Due to fundamental changes, there is not a direct upgrade path. The purpose of this documentation is to provide an end-to-end guide for porting your EF6 applications to EF Core.
 
 > [!IMPORTANT]
 > Before you start the porting process it is important to validate that EF Core meets the data access requirements for your application.
 
-## Missing features
+## When to port
 
-Make sure that EF Core has all the features you need to use in your application. See [Feature Comparison](xref:efcore-and-ef6/index) for a detailed comparison of how the feature set in EF Core compares to EF6. If any required features are missing, ensure that you can compensate for the lack of these features before porting to EF Core.
+It is not a requirement that you port your code to EF Core. There are many reasons why you may choose to remain on EF6. The latest version of EF6 supports .NET Core. If your application is running fine and you don't need any of the latest features, you may wish to consider remaining on EF6. Some reasons you might consider porting:
 
-## Behavior changes
+- Take advantage of the ongoing performance improvements in EF Core. For example, one customer who migrated from EF6 to EF Core 6 saw a 40x reduction in use of a heavy query due to the [query splitting feature](/ef/core/querying/single-split-queries/). Many customers report enormous performance gains simply by moving to the latest EF Core.
 
-This is a non-exhaustive list of some changes in behavior between EF6 and EF Core. It is important to keep these in mind as your port your application as they may change the way your application behaves, but will not show up as compilation errors after swapping to EF Core.
+- Use new features in EF Core. There will be no new features added to EF6. All of the new functionality, for example the [Azure Cosmos DB provider](/ef/core/providers/cosmos/) and [`DbContextFactory`](/ef/core/what-is-new/ef-core-5.0/whatsnew#dbcontextfactory), will only be added EF Core.
```

Comment:
We should update and link to the things that EF Core supports that EF6 does not. (And vice-versa, but calling this out here because many people seem to be unaware of the many fundamental things that are build into EF Core but will never be supported in EF6; example relationships using alternate keys.)